app/controllers/api/v1/worksheets_controller.rb

```diff
@@ -20,6 +20,9 @@ def update
     image_url = remote_image_url
     prepped_params = worksheet_params.except(:image_url, :photo).merge({ image_url: image_url })
     if @worksheet.update(prepped_params)
+      students = @worksheet.work_group.students.to_a
```

Comment:
better solution would be `@worksheet.work_group.students.pluck(:email)`
this will pull out the emails directly in an array so you don't need to transform to array and then map it